README.md

```diff
@@ -2,6 +2,13 @@
 
 Gradle plugin for creating fat/uber JARs with support for package relocation.
 
+> [!NOTE]\
```

Comment:
This is a GitHub "alert" section, see [documentation](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts).